src/v2/guide/team.md

```diff
@@ -5,6 +5,7 @@ order: 803
 ---
 
 {% raw %}
+
```

Comment:
Let's remove the changes for the whole file here

Reply:
I guess my contribution was not merged!?

Reply:
yeah, at the end https://github.com/vuejs/vuejs.org/pull/2353 got merged